Dockerfile

```diff
@@ -16,14 +16,14 @@ RUN apt-get update \
 RUN mkdir -p /opt/app
 WORKDIR /opt/app
 
+RUN npm install -g corepack
```

Comment:
We want a specfic version of corepack since last time it broke everything

Reply:
Can actually replace it with `corepack enable` and just use the one from the image?